rentsafe/rent_views/new_clients/forecasts.py

```diff
@@ -7,7 +8,7 @@
 def get_forecast_inflows(request):
     current_date = datetime.now().date()
     forecast_inflows_list = []
-    current_rate = LeaseCurrencyRate.objects.filter(company_id=request.user.company).first()
+    current_rate = CurrencyRate.objects.filter(user=request.user).first()
```

Comment:
Stick to one desired logic:

```suggestion
    current_rate = CurrencyRate.objects.filter(user__company=request.user.company).first()
```